frontend/src/components/pages/rp-connect/Pipelines.Create.tsx

```diff
@@ -125,7 +125,11 @@ class RpConnectPipelinesCreate extends PageComponent<{}> {
               width={500}
             />
           </FormField>
-          <FormField label="Tasks">
+          <FormField
+            label="Tasks"
+            description="One task is equivalent to 0.1 CPU and 400 MB of memory. This is enough to experiment with low-volume pipelines. For pipelines that include AI components, one task is equivalent to 1 GPU. This can have cost implications."
```

Comment:
would it help to make it an info alert or was it just meant to be a description for the form field